equalsverifier-core/src/test/java/nl/jqno/equalsverifier/integration/operational/MultipleTypeEqualsVerifierTest.java

```diff
@@ -241,7 +241,7 @@ void succeed_whenCallingForPackageRecursivelyOnAPackageContainingFailingClasses_
     void succeed_whenReportingOnSeveralCorrectClasses() {
         List<EqualsVerifierReport> reports = EqualsVerifier.forClasses(A.class, B.class, C.class).report();
 
-        assertThat(reports.size()).isEqualTo(3);
+        assertThat(reports).hasSize(3);
```

Comment:
Now when this fails you'll get a message saying "a collection was expected with three elements, but it was a collection with x elements, containing elem1, elem2, ....". Much more helpful than expected 3 but was 5.